packages/api-generator/src/locale/en/group.json

```diff
@@ -0,0 +1,12 @@
+{
+  "props": {
+    "modelValue": "Represents the selected item(s). Will be an array if **multiple** prop is used.",
+    "mandatory": "Forces at least one item to always be selected (if available).",
+    "max": "Sets a maximum number of selections that can be made.",
```

Comment:
What about a min prop?

Reply:
How would `min` work? Say it's set to 2. I select one item. What happens? Is it shown as selected just as if min was not set? Are we emitting new value with it included? Probably not right, because 1 is less than 2. That means we now have to create another internal value that differs from the 'real' value to keep track of 'selected but not valid' items.

If `min` is only a validation thing like for forms, not a hard limit like `max`, then it should probably not live in this component.